src/main/java/com/woowacamp/soolsool/core/statistics/controller/StatisticsController.java

```diff
@@ -29,4 +36,26 @@ public ResponseEntity<ApiResponse<Void>> updateStatisticsSales() {
 
         return ResponseEntity.ok().build();
     }
+
+    @NoAuth
+    @RequestLogging
+    @GetMapping("/price")
+    public ResponseEntity<ApiResponse<List<LiquorSalePriceResponse>>> findTop5SalePrice() {
+        final List<LiquorSalePriceResponse> liquorSalePriceResponses
+            = statisticsService.findTop5LiquorsBySalePrice();
```

Comment:
```suggestion
        final List<LiquorSalePriceResponse> liquorSalePriceResponses =
            statisticsService.findTop5LiquorsBySalePrice();
```

고치라는 건 아니지만 저는 이렇게 사용하긴 합니다! 신기하네요.

Reply:
chaining method 쓸 때 온점 (.) 을 다음 행에 써서 자연스럽게 저렇게 썼는데 보통 어떻게 쓰는지 궁금하긴 하네요🧐